src/types.ts

```diff
@@ -1,14 +1,42 @@
 import * as azdata from 'azdata';
 
+/**
+ * Contains success information, a sessionId to be used when requesting
```

Comment:
comment mentions values that don't appear to be included in the interface definition (e.g. success info & root node).

Reply:
See here: https://github.com/microsoft/sqlops-dataprotocolclient/pull/75#discussion_r1046273032